app/js/models/todo.js

```diff
@@ -4,7 +4,8 @@ define([
   return Backbone.Model.extend({
     defaults: {
       name: '',
-      done: 0
+      done: false,
+      removed: false
```

Comment:
We need a test for this model to check the default values for the model properties.